lib/AST/Bridging/DeclBridging.cpp

```diff
@@ -117,6 +147,15 @@ void BridgedDecl_forEachDeclToHoist(BridgedDecl cDecl,
   });
 }
 
+BridgedDeclContext BridgedDecl_getDeclContext(BridgedDecl decl) {
```

Comment:
do you need this?

Reply:
As I said in my other comment, the best solution is to make `DeclContext` a protocol